packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -492,13 +492,40 @@ mixin WidgetsBinding
       _debugShowWidgetInspectorOverrideNotifierObject ??= ValueNotifier<bool>(false);
   ValueNotifier<bool>? _debugShowWidgetInspectorOverrideNotifierObject;
 
+  /// The default behavior for taps in the app when the widget inspector is
+  /// enabled.
+  ///
+  /// - If true, taps in the app are intercepted by the widget inspector.
+  /// - If false, taps in the app are not intercepted by the widget inspector.
+  bool get debugWidgetInspectorDefaultTapBehaviorEnabled {
```

Comment:
should this name provide information as to what the default behavior is? for example, debugWidgetInspectorInterceptTapsEnabled or something similar? As is, you'd have to go to the definition to understand what the default behavior is.

Reply:
Good point, renamed to `debugWidgetInspectorSelectionOnTapEnabled`